### ISSUE TITLE
Delay JavaScript Execution compatibility with Bookingsuedtirol.com

### ISSUE THREAD

**Issue by DahmaniAdame** [wp-media/wp-rocket#4599]
**Before submitting an issue please check that you’ve completed the following steps:**
- [x] Made sure you’re on the latest version `3.10.5`
- [x] Used the search feature to ensure that the bug hasn’t been reported before

**Describe the bug**
[Bookingsuedtirol.com](https://www.bookingsuedtirol.com/) has a widget that can be inserted in any website. 

```
<div id="Bs-SpecialsWidget"></div>
<script id="booking-suedtirol-js" src="https://widget.bookingsuedtirol.com/v2/bundle.js"></script>
<script>
    document
    .querySelector("#booking-suedtirol-js")
    .addEventListener("load", function() {
        XYZ.Widgets.Specials("#Bs-SpecialsWidget", {
            accommodationId: #####,
            lang: 'en',
          	layout: 'layout-5'
        });
    });
</script>
```
When used with Delay JavaScript Execution, the widget's content doesn't show up even after user interaction. 

The issue is fixed by excluding: 

```
 booking-suedtirol-js
```

No error showing on the console.

**To Reproduce**
1. Insert the widget
2. Enable Delay JavaScript Execution
3. See the missing widget content (might require valid account)

**Expected behavior**
The widget should at least work with Delay JavaScript Execution after user interaction.

**Screenshots**
N/A

**Additional context**
Ticket - https://secure.helpscout.net/conversation/1727751196/314385/

Added to https://docs.wp-rocket.me/article/1560-delay-javascript-execution-compatibility-exclusions#bookingsuedtirol

**Backlog Grooming (for WP Media dev team use only)**
- [ ] Reproduce the problem
- [ ] Identify the root cause
- [ ] Scope a solution
- [ ] Estimate the effort

**Comment by GeekPress** [comment]
@DahmaniAdame Can you please do a PR including the patterns you added into the doc to our default exclusion list? Please check with a developer if you don't know where to add the patterns.

**Comment by remyperona** [comment]
Based on the provided code snippet, we could use `booking-suedtirol-js` to exclude both at the same time

**Comment by DahmaniAdame** [comment]
@GeekPress will do 👍
I believe the exclusions go here - https://github.com/wp-media/wp-rocket/blob/cfebe7a459e1106171ccb5d1eade107ece81b6eb/inc/Engine/Optimization/DelayJS/HTML.php#L26-L68
@Tabrisrp just to be sure, can you please confirm that's the right file before I proceed? 🙏

**Comment by vmanthos** [comment]
> can you please confirm that's the right file before I proceed?

@DahmaniAdame You got this right. 👍

**Comment by iCaspar** [comment]
Closed by #4604
